src/api/config/routes/webui.rb

```diff
@@ -218,7 +218,6 @@
     post 'project/restore' => :restore, constraints: cons, as: 'projects_restore'
     patch 'project/update' => :update, constraints: cons
     delete 'project/destroy' => :destroy
-    get 'project/requests/:project' => :requests, constraints: cons, as: 'project_requests'
```

Comment:
This has to stay, considering that the feature is in beta and we aren't just removing it for everyone

Reply:
Ah right.